code/game/objects/items/weapons/tools.dm

```diff
@@ -268,7 +268,7 @@ obj/item/weapon/wrench/power/suicide_act(mob/user)
 	toolspeed = 4
 
 /obj/item/weapon/wirecutters/power/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is [src] around [user.p_their()] neck. It looks like [user.p_theyre()] trying to rip [user.p_their()] head off!</span>")
+	user.visible_message("<span class='suicide'>[user] is wrapping \the [src] around [user.p_their()] neck. It looks like [user.p_theyre()] trying to rip [user.p_their()] head off!</span>")

```

Comment:
or not capitalized if it's not actually proper, because we near-universally don't capitalize items